packages/dev/sharedUiComponents/src/fluent/hoc/propertyLines/vectorPropertyLine.tsx

```diff
@@ -177,8 +181,7 @@ export const QuaternionPropertyLine: FunctionComponent<QuaternionPropertyLinePro
 
     const onEulerChange = (val: Vector3) => {
         const quat = Quaternion.FromEulerAngles(val.x, val.y, val.z);
-        setQuat(quat);

```

Comment:
Do we really want to remove this? Doesn't this mean we are relying on the changed value flowing back into this component? I thought elsewhere we proactively updated the state directly. Or is there an issue with two sources of truth fighting to set the value?